server/odc-service/src/main/java/com/oceanbase/odc/service/session/util/DBSchemaExtractor.java

```diff
@@ -420,6 +421,18 @@ public RelationFactor visitPl_schema_name(Pl_schema_nameContext ctx) {
             return null;
         }
 
+        @Override
+        public RelationFactor visitSql_stmt(Sql_stmtContext ctx) {
+            try {
+                OBOracleRelationFactorVisitor visitor = new OBOracleRelationFactorVisitor();
+                visitor.visit(ctx.getChild(0).getChild(0));
```

Comment:
is there any possible that ctx.getChild(0) is null or ctx.getChild() is zero length?

Reply:
yes, you are right, I have changed it~